frontend/src/components/elements/Alert/Alert.tsx

```diff
@@ -46,13 +47,15 @@ export interface AlertProps {
  * Display an (error|warning|info|success) box with a Markdown-formatted body.
  */
 export default function Alert({
+  icon,
   body,
   kind,
   width,
 }: AlertProps): ReactElement {
   return (
     <div className="stAlert">
       <AlertContainer width={width} kind={kind}>
+        {icon && <StreamlitMarkdown source={icon} allowHTML={false} />}
```

Comment:
Can we use our <Icon> component here?

Reply:
Just to make sure sizes and such are the same as our icons.

Reply:
Done! Updated on [fa4d237](https://github.com/streamlit/streamlit/pull/4964/commits/fa4d237dba9e8b082cddf26a39bf6bccbeaa9a02)